### PR TITLE
Reworked, added test, babel

### PR DESCRIPTION
**Pull request by AntonBryansky**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,31 @@
-node_modules
-.vscode/
-config.json
\ No newline at end of file
+# Folders #
+###########
+_obj
+node_modules/
+
```
Comment (ichi404gh) [inline .gitignore:5]: Please, keep ignoring my `.vscode/` folder.

File: package.json [hunk 2]
```diff
@@ -4,13 +4,20 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node index.js"
+    "build": "babel src -d lib",
+    "test": "mocha --compilers js:babel-core/register",
+    "start": "babel-node index.js"
```
Comment (ichi404gh) [inline package.json:9]: you main file is `src/index.js` (or `lib/...`), doesn't it?

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+import TelegramBot from 'node-telegram-bot-api';
+// Import
+import config from './config';
+// Import filter functions
+import {
+  isJoinedMessage,
+  isArabicMessage,
+  isUrlMessage
+} from './filters.js'
+
```
Comment (ichi404gh) [inline src/index.js]: Your file is `filter.js`, not `filters.js`

**Review by ichi404gh — CHANGES_REQUESTED** [review]
Please make changes in accordance with the comments.
And btw, I committed more filters recently, please include it in your pr.

File: src/index.js (hunk 3 above)
Comment (AntonBryansky) [reply src/index.js]: At now `filters.js`

File: package.json (hunk 2 above)
Comment (AntonBryansky) [reply package.json:9]: main files at build folder after `npm build`. After it will at `./buils/index.js`

**Pull request closed without merging**